integreat_cms/cms/migrations/0079_alter_page_mirrored_page.py

```diff
@@ -0,0 +1,52 @@
+# Generated by Django 3.2.20 on 2023-09-07 10:59
+
+from cacheops import invalidate_model
+from django.db import migrations
+
+from ..constants import region_status
+
+
+# pylint: disable=unused-argument
+def update_mirrored_pages(apps, schema_editor):
+    """
+    Set the field mirrored_page to None when a page is archived or belongs to an archived region.
+
+    :param apps: The configuration of installed applications
+    :type apps: ~django.apps.registry.Apps
+
+    :param schema_editor: The database abstraction layer that creates actual SQL code
+    :type schema_editor: ~django.db.backends.base.schema.BaseDatabaseSchemaEditor
+    """
+
+    Page = apps.get_model("cms", "Page")
+    Region = apps.get_model("cms", "Region")
+
+    pages_archived_region = Page.objects.filter(region__status=region_status.ARCHIVED)
+    invalidate_model(Region)
```

Comment:
```suggestion
```

This probably needs to be moved to the end of the function to take effect, right?
I merged the `pages_archived_region` pages with the archived pages later to do the update in one combined query.